e2e/features/Udacity.feature

```diff
@@ -33,4 +39,22 @@ Scenario: 3. The card details, dropdown filter and filter fields should work cor
    Then the selected filters field should contain the following filters:
         |Intermediate|
     And the intermediate course level logo should be visible for all cards
-    And the "Intermediate" course level label should be visible for all cards
\ No newline at end of file
+    And the "Intermediate" course level label should be visible for all cards
+
+
+Scenario: 4. The LEARN MORE button and short description should be visible
+  Given the Udacity course page is opened
+
+   When the text "android" is typed into the search bar
+    And the "Android Basics by Google" card's details expander is clicked
```

Comment:
It would be better: the "Android Basics by Google" card's details expander is opened